cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/graph/scopes/ValueDeclarationScope.kt

```diff
@@ -40,10 +40,10 @@ import org.slf4j.LoggerFactory
  * Works for if, for, and extends to the block scope
  */
 open class ValueDeclarationScope(override var astNode: Node?) : Scope(astNode) {
-    var valueDeclarations = mutableListOf<ValueDeclaration>()
+    @Transient var valueDeclarations = mutableListOf<ValueDeclaration>()
```

Comment:
Why are we not persisting these? Just to catch me up on what in the scopes we want to store in a graph and what not

Reply:
Ok, we could actually persist those, I just didn't want to add too many edges. The other transient properties are Maps, which cannot be persisted without major pain.

Do you want me to remove `@Transient` from the declarations list in `ValueDeclarationScope` and `StructureDeclarationScope`?

Reply:
No, if the assesment is that thats to many edges to persist into the neo4j I am fine with that. we can change our strategy in the future if we want.